tests/cli/commands/test_scheduler_command.py

```diff
@@ -74,6 +74,38 @@ def test_skip_serve_logs(self, mock_process, mock_scheduler_job, executor):
             with pytest.raises(AssertionError):
                 mock_process.assert_has_calls([mock.call(target=serve_logs)])
 
+    @mock.patch("airflow.utils.db.check_and_run_migrations")
+    @mock.patch("airflow.utils.db.synchronize_log_template")
+    @mock.patch("airflow.cli.commands.scheduler_command.SchedulerJob")
+    @mock.patch("airflow.cli.commands.scheduler_command.Process")
+    @pytest.mark.parametrize("executor", ["LocalExecutor"])
+    def test_check_migrations_is_false(
+        self, mock_process, mock_scheduler_job, mock_log, mock_run_migration, executor
+    ):
+        args = self.parser.parse_args(["scheduler"])
+        with conf_vars(
+            {("core", "executor"): "SequentialExecutor", ("database", "check_migrations"): "False"}
```

Comment:
```suggestion
            {("database", "check_migrations"): "False"}
```

Is the executor config relevant here?

Reply:
Thanks, @jedcunningham , just cleaned up!